src/search/sr_optimizers.c

```
@@ -0,0 +1,331 @@
+#include "search_optimizer.h"
+
+typedef int (*sr_optimizer_run_fn)(int ndim, real dpos, const char *bak_file,
+                                   const char *log_file);
+
+typedef struct sr_optimizer_entry {
+  sr_optimizer_def def;
+  const char *aliases[8];
+  sr_optimizer_run_fn run;
+} sr_optimizer_entry;
+
+static int sr_run_simplex(int ndim, real dpos, const char *bak_file,
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'sr_run_simplex' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)